code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1312,5 +1319,151 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/trophazole
+	name = "Trophazole"
+	description = "Orginally developed as fitness supplement, this chemical accelerates wound healing and if ingested turns nutriment into healing peptides"
+	reagent_state = LIQUID
+	color = "#FFFF6B"
+	overdose_threshold = 20
+
+/datum/reagent/medicine/trophazole/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-1.5*REM, 0.) // heals 3 brute & 0,5 burn if taken with food. compared to 2.5 brute from bicard + nutriment
+	..()
+	. = 1
+
+/datum/reagent/medicine/trophazole/overdose_process(mob/living/M)
+	M.adjustBruteLoss(3*REM, 0)
+	..()
+	. = 1
+
+/datum/reagent/medicine/trophazole/reaction_mob(mob/living/carbon/M, method=INGEST, reac_volume)
+	if(method != INGEST)
+		return
+
+	M.reagents.remove_reagent(/datum/reagent/medicine/trophazole/, reac_volume * 0.05)
```

Comment:
is there supposed to be a slash at the end of trophazole in this?

Reply:
probably not